include/deal.II/grid/grid_tools.h

```diff
@@ -606,6 +606,38 @@ namespace GridTools
   /*@{*/
 
   /**
+   * Given a @p cache and a list of @p points create the quadrature rules. This
+   * function returns a tuple containing the following elements:
+   * - The first ( get<0>), call it cells, is a vector of the cells containing at least one of
+   *  the @p points
+   * - The second, call it qpoints, is a vector of vectors. Every vector corresponds to the
+   *  cell of cells with the same index and contains the map to the unit cell of all elements
+   *  of points inside this cell
+   * - The third, call it maps, contains the index of the given quadrature points, i.e.,
```

Comment:
- a vector `indices` of vector of integers, containing the mapping between local numbering in `qpoints`, and global index in `points`

If `points[a]` and `points[b]` are the only two points that fall in `cells[c]`, then `qpoints[c][0]` and `qpoints[c][1]` are the reference positions of `points[a]` and `points[b]` in `cells[c]`, and `indices[c][0] = a`, `indices[c][1] = b`. The function `Mapping::tansform_unit_to_real(qpoints[c][0])` returns `points[a]`.